website/docs/reference/ecosystem/index.mdx

```diff
@@ -20,6 +20,6 @@ sidebar_position: 5
 
 ![Lade demo](https://github.com/zifeo/lade/blob/main/examples/demo.gif?raw=true)
 
-## Python WASI reactor
+## Ghjk 
```

Comment:
👍 can you also add to ghjk the same banner as we have in whiz/lade that advertise Metatype please?